superset/assets/src/dashboard/stylesheets/filter-scope-selector.less

```diff
@@ -108,6 +131,10 @@
     }
 
     .react-checkbox-tree {
+      .rct-title .root {
+        font-weight: bold;
```

Comment:
i see here we're using `font-weight: bold;` but below it's `font-weight: 700;`. Can we standardize?

Reply:
i choose to use `bold`.

Reply:
Not to sidetrack the discussion, but I'm considering if it's worth moving some of these values (and other related typographic treatments) to LESS variables, so we can use `font-weight: @bold` and set/theme that to be 700 (or whatever!) globally.